README.md

```
@@ -8,31 +8,25 @@
 [![discord](https://img.shields.io/badge/join-bibliothecadao-black?logo=discord&logoColor=white)](https://discord.gg/realmsworld)
 [![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
 
-![background](/apps/docs/assets/images/bg.png)
+# Realms.World Account Portal
 
-# Realms.World Account Portal 
-
-### The account potal 
+### The account potal
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)